README.md

```diff
@@ -467,6 +468,13 @@ var signIn = new OktaSignIn(config);
     logo: 'https://acme.com/img/logo.png'
     ```
 
+- **logoText:** Text for `alt` attribute of the logo image
```

Comment:
this is only when image is not displayed. Make it clear in the description or comments.